internal/pkg/skuba/upgrade/node/versions.go

```diff
@@ -35,6 +35,17 @@ type NodeVersionInfoUpdate struct {
 	Update  kubernetes.NodeVersionInfo
 }
 
+func (nviu NodeVersionInfoUpdate) HasMajorOrMinorUpdate() bool {
+	if nviu.Current.IsControlPlane() {
+		if nviu.Update.APIServerVersion.Major() > nviu.Current.APIServerVersion.Major() ||
+			nviu.Update.APIServerVersion.Minor() > nviu.Current.APIServerVersion.Minor() {
```

Comment:
it's probably a very minor enhancement but as the major updates are very infrequent, we could check for minor first to speed up the check a little bit

Reply:
We can change that later, however that would mean negative logic on the pattern installation (`if !isPatchUpdate { installIntermediatePattern }`).

Reply:
it shouldn't change any outside logic i meant like this:

```go
if nviu.Update.APIServerVersion.Minor() > nviu.Current.APIServerVersion.Minor() ||
        nviu.Update.APIServerVersion.Major() > nviu.Current.APIServerVersion.Major()
```

Reply:
Ah misunderstood. Did it in level of importance but yes I don't even know if Kubernetes 2.0 is planned :)